master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -242,6 +243,28 @@ func (b BindMountsConfigV0) Merge(other interface{}) interface{} {
 	return out
 }
 
+// EntrypointV0 configures the entrypoint script for the experiment
+type EntrypointV0 struct {
+	RawEntrypoint []string
+}
+
+// MarshalJSON marshals makes the EntrypointV0 container transparent to marshaling.
+func (d EntrypointV0) MarshalJSON() ([]byte, error) {
+	return json.Marshal(d.RawEntrypoint)
+}
+
+// UnmarshalJSON implements the json.Unmarshaler interface.
+func (e *EntrypointV0) UnmarshalJSON(data []byte) error {
+	var entrypoint []string
+	if err := json.Unmarshal(data, &entrypoint); err == nil {
+		e.RawEntrypoint = []string{}
+		e.RawEntrypoint = append(e.RawEntrypoint, entrypoint...)
+		return nil
+	}
+	e.RawEntrypoint = strings.Fields(string(data))
```

Comment:
Maybe just leave the string in its original format?  I think the expected behavior here is for the single-string case to be interpreted as `sh -c $the_string`, which means that you can't really parse it here.

Also, the master doesn't really care what is inside of this struct anyway, so maybe you could get away with having it be an `interface{}`?  Not sure if that will lead to complications or not.

Reply:
done